api/src/main/resources/liquibase-update-to-2.2.xml

```diff
@@ -117,14 +117,29 @@
                                      baseTableName="patient_program_attribute" baseColumnNames="voided_by"
             referencedTableName="users" referencedColumnNames="user_id" />
     </changeSet>
+	<changeSet id="20180501-TRUNK-5338" author="Kelechi+iheanyi">
+		<preConditions onFail="MARK_RAN">
+			<tableExists tableName="conditions"/>
+		</preConditions>
+		<comment>
+			Modify conditions table to match condition object in the core platform if the conditions table already exists in the database
+		</comment>
+		<renameColumn columnDataType="int" newColumnName="previous_version" oldColumnName="previous_condition_id" tableName="conditions"/>
```

Comment:
I would prefer to have these as separate changesets with their respective comments being explicit about what they do e.g **Rename tableA.x column to y**

Reply:
Okay. Done